spring-integration-jdbc/src/main/resources/org/springframework/integration/jdbc/schema-db2.sql

```diff
@@ -46,12 +46,11 @@ CREATE TABLE INT_CHANNEL_MESSAGE (
 	MESSAGE_SEQUENCE BIGINT NOT NULL ,
 	MESSAGE_BYTES BLOB,
 	REGION VARCHAR(100) NOT NULL,
-	constraint INT_CHANNEL_MESSAGE_PK primary key (REGION, GROUP_KEY, CREATED_DATE, MESSAGE_SEQUENCE)
+	constraint INT_CHANNEL_MESSAGE_PK primary key (GROUP_KEY, MESSAGE_ID, REGION)
 );
 
-CREATE INDEX INT_CHANNEL_MSG_DELETE_IDX ON INT_CHANNEL_MESSAGE (REGION, GROUP_KEY, MESSAGE_ID);
--- This is only needed if the message group store property 'priorityEnabled' is true
--- CREATE UNIQUE INDEX INT_CHANNEL_MSG_PRIORITY_IDX ON INT_CHANNEL_MESSAGE (REGION, GROUP_KEY, MESSAGE_PRIORITY DESC, CREATED_DATE, MESSAGE_SEQUENCE);
+CREATE INDEX INT_CHANNEL_MSG_DATE_IDX ON INT_CHANNEL_MESSAGE (CREATED_DATE, MESSAGE_SEQUENCE);
+CREATE INDEX INT_CHANNEL_MSG_PRIORITY_IDX ON INT_CHANNEL_MESSAGE (MESSAGE_PRIORITY DESC, CREATED_DATE, MESSAGE_SEQUENCE);
```

Comment:
Why changes to this table here, but not on master?

Reply:
Oh! this must not be here. The problem is with `vpp` .We Don't have it on master any more.
Will fix it later here.
We must keep your index here.